### PR TITLE
font-awesome: new package at 5.12.1

### PR DESCRIPTION
**Pull request by eudaldgr**
## Description of package

[Font Awesome 5](https://github.com/FortAwesome/Font-Awesome)

## Installed manifest of package

```
/var/db/kiss/installed/font-awesome/version
/var/db/kiss/installed/font-awesome/sources
/var/db/kiss/installed/font-awesome/manifest
/var/db/kiss/installed/font-awesome/checksums
/var/db/kiss/installed/font-awesome/build
/var/db/kiss/installed/font-awesome/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/fonts/OTF/Font Awesome 5 Free-Solid-900.otf
/usr/share/fonts/OTF/Font Awesome 5 Free-Regular-400.otf
/usr/share/fonts/OTF/Font Awesome 5 Brands-Regular-400.otf
/usr/share/fonts/OTF/
/usr/share/fonts/
/usr/share/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/font-awesome/build [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+
+install -Dm644 ./otfs/*.otf -t "$1/usr/share/fonts/OTF"
```
Comment (dylanaraps) [inline community/font-awesome/build]: The `-t` flag for `install` isn't very portable. This also works:

```
mkdir -p "$1/usr/share/fonts/OTF"
cp ./otfs/*.otf "$1/usr/share/fonts/OTF"
```

File: community/font-awesome/sources [hunk 2]
```diff
@@ -0,0 +1 @@
+https://github.com/FortAwesome/Font-Awesome/archive/5.12.1.tar.gz
```
Comment (dylanaraps) [inline community/font-awesome/sources]:
```
FortAwesome
```

Typo? Did you not build this package before submitting?

**Comment by dylanaraps** [comment]
```
/var/db/kiss/installed/font-awesome/depends
```

Why is this in the manifest if there are no dependencies?

File: community/font-awesome/sources (hunk 2 above)
Comment (eudaldgr) [reply community/font-awesome/sources]: I build the package before submitting. Did you check the link before commenting? Now seriously, the source link is correct.

**Comment by eudaldgr** [comment]
> ```
> /var/db/kiss/installed/font-awesome/depends
> ```
> 
> 
> Why is this in the manifest if there are no dependencies?

I don't know, I'm going to look at what has happened.

File: community/font-awesome/sources (hunk 2 above)
Comment (dylanaraps) [reply community/font-awesome/sources]: Well, that's unexpected. :woman_shrugging:

**Comment by eudaldgr** [comment]
Solved was a previously depend, that it is no longer necessary.

File: community/font-awesome/build (hunk 1 above)
Comment (eudaldgr) [reply community/font-awesome/build]: Without `install -Dm644` command?

Comment (dylanaraps) [reply community/font-awesome/build]: Yes

Comment (eudaldgr) [reply community/font-awesome/build]: Done

**Comment by dylanaraps** [comment]
Thanks

**Pull request merged**